### PR TITLE
Tweak styles for iOS smart banner and add styles for Android smart banner

### PR DESCRIPTION
**Pull request by theonicolaou**

### CODE REVIEW

**Comment by theonicolaou** [comment]
@ain I have made changes as per the SCSS lint rules

File: src/smartbanner.scss [hunk 1]
```diff
@@ -4,7 +4,7 @@ $blue: #1474fc;
 
 .smartbanner {
   position: absolute;
-  top: 0;
+  top: -84px;
```
Comment (ain) [inline src/smartbanner.scss]: We can't make changes to existing styles, that will break it for the current users. 

Let's please keep all the current styles intact and make Android-related changes in `smartbanner--android`.

Comment (theonicolaou) [inline src/smartbanner.scss]: @ain Ok, understood re: default styles. I have reinstated them and kept the Android styles.

**Comment by theonicolaou** [comment]
@ain I believe the build for this PR is passing now - is it ok to merge?

**Comment by ain** [comment]
@theonicolaou thanks for a reminder. I also wanted to give it a final test before merging. Hopefully I have time for this tomorrow evening. 

If all is fine, we can also do a direct release.

**Comment by ain** [comment]
@theonicolaou I have tested the Android and I can not see a difference. Looks the same as iOS.

Am I overlooking something?

**Comment by theonicolaou** [comment]
@ain can you see styles in the dist/css file for android?

**Comment by theonicolaou** [comment]
Hi @ain, any update on if/when my PR will be merged? Is there anything else I need to do for this?

**Comment by ain** [comment]
Thanks for the work @theonicolaou, I've retested, all worked as expected. 

I'll release `1.7.0` with it this week.

**Comment by ain** [comment]
https://github.com/ain/smartbanner.js/releases/tag/v1.7.0

**Pull request merged**